<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novocontato</class>
 <widget class="QDialog" name="fm_novocontato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leNome"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Telefone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTelefone"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEmail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btGravar">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btCancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
